apps/perf-test/tasks/perf-test.js

```diff
@@ -188,7 +189,11 @@ const urlForDeployPath = process.env.BUILD_SOURCEBRANCH
   ? `http://fabricweb.z5.web.core.windows.net/pr-deploy-site/${process.env.BUILD_SOURCEBRANCH}/perf-test`
   : 'file://' + path.resolve(__dirname, '../dist/');
 
-const urlForDeploy = urlForDeployPath + '/index.html';
+// Temporarily comment out deploy site usage to speed up CI build time and support parallelization.
+// At some point perf test should be broken out from CI default pipeline entirely and then can go back to using deploy site.
+// For now, use local perf-test bundle so that perf-test job can run ASAP instead of waiting for the perf-test bundle to be deployed.
+// const urlForDeploy = urlForDeployPath + '/index.html';
+const urlForDeploy = 'file://' + path.resolve(__dirname, '../dist/') + '/index.html';
```

Comment:
Does this need to be `file:///`? And does it work on both Mac and Windows?

Reply:
Per my comments, I need to do this in order to prevent PerfTest from being a super long pole. If I don't it also needs to wait for the `Deploy` job to complete, which pushes total build time past 30 minutes (same as master), negating all of the perf gains. This is the same URL that has been used when running perf-test locally all along for both Mac and Windows (see line 189.)

Reply:
resolve will add that `/` for you